woocommerce/woocommerce-gutenberg-products-block
# Migrate Cart i1 to Cart i2

This PR migrates Cart i1 to Cart i2.

Closes #4798

### Testing steps

Migration process:

1. Before using this PR, create a page containing a cart block, change alignment.
2. Use this PR. Load the page in the editor.
3. Confirm it shows the correct content in both filled/empty states. Do not save.
4. View frontend - confirm cart is functional.
5. Save page in editor.
6. View frontend - confirm cart is functional.

Cart in general:

1. Insert cart into page and smoke test functionality.

Mini Cart:

1. Insert mini cart and smoke test functionality.

### Dev note

> Some Cart block options might not work in the frontend unless you resave the cart block again in the editor.

### Changelog

> Cart v2: The cart block, like checkout block, now supports inner blocks that allow for greater customizability.
